src/Symfony/Bundle/TwigBundle/Tests/Loader/FilesystemLoaderTest.php

```diff
@@ -98,7 +98,7 @@ public function testTwigErrorIfLocatorReturnsFalse()
     public function testTwigErrorIfTemplateDoesNotExist()
     {
         $this->expectException('Twig\Error\LoaderError');
-        $this->expectExceptionMessageRegExp('/Unable to find template "name\.format\.engine" \(looked into: .*Tests.Loader.\.\..DependencyInjection.Fixtures.Resources.views\)/');
+        $this->expectExceptionMessageMatches('/Unable to find template "name\.format\.engine" \(looked into: .*Tests.Loader.\.\..DependencyInjection.Fixtures.Resources.views\)/');
```

Comment:
💡 This method has been renamed in PHPUnit 9. PhpUnitBridge already polyfills the new method.